HelpSource/Classes/DrawGridX.schelp

```diff
@@ -0,0 +1,28 @@
+CLASS:: DrawGridX
+summary:: A sub-method of DrawGrid, used for setting up its horizontal size
+categories:: GUI>Accessories
+related:: Reference/plot, Classes/GridLines, Classes/Plotter, Classes/UserView, Classes/DrawGrid
+
+DESCRIPTION::
+DrawGridX, is a method instantiated by DrawGrid Class, in its init functon, to set the horizontal size of a GridUI Element, just as seen in the code below
+
+CLASSMETHODS::
+
+METHOD:: copy
+used for copying this instance
+
+EXAMPLES::
+
+code::
+// taken from Grid.sc class file
+init { arg bounds,h,v;
```

Comment:
Examples should be code that the user can execute directly. The only time an example should be class-library code is if the help file is about writing classes. I disagree with this example.